guides/v2.2/cloud/project/project-conf-files_services-rabbit.md

```diff
@@ -72,6 +72,42 @@ You can do this using [SSH tunneling]({{ page.baseurl }}/cloud/env/environments-
 
 1.  While the session is open, you can start a RabbitMQ client of your choice from your local workstation, configured to connect to the `localhost:<portnumber` using the username and password you found in the relationship variable. For this example, you would use `localhost:5672`.
 
+### Get RabbitMQ credentials {#cloud-rabbitmq-credentials}
+
+We use the {{site.data.var.ece}} environment variable [`$MAGENTO_CLOUD_RELATIONSHIPS`]({{ page.baseurl }}/cloud/env/environment-vars_cloud.html), a JSON object, to retrieve environment-related relationships.
+
+To verify this information used for configurations and settings:
+
+1. SSH into the Integration environment with RabbitMQ installed and configured.
```

Comment:
Please, adjust this part to make the explanation more unified for all environments
```suggestion
1. SSH into your Magento Cloud environment where RabbitMQ is installed and configured.
```

Reply:
**implemented**

Thank you, @rogyar